vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -77,7 +78,11 @@ func main() {
 	metrics.Initialize(*address, healthCheck)
 	metrics_admission.Register()
 
-	certs := initCerts(*certsConfiguration)
+	// load certs
+	kpr, err := NewKeypairReloader(*certsConfiguration)
+	if err != nil {
+		klog.Fatal(err)
```

Comment:
More info in the log needed, like "Failed to load certificate on startup: %v", err